andino_bringup/config/raspicam.yaml

```diff
@@ -0,0 +1,27 @@
+image_width: 640
```

Comment:
Can you provide some links to documentation on how you can calibrate/obtain the intrinsic of your camera?
Add this as a comment in the yaml file.